.github/actions/setup-argocd-image-updater/action.yml

```diff
@@ -0,0 +1,19 @@
+name: Setup ArgoCD Image Updater
+description: Install ArgoCD Image Updater and add it to the PATH
+inputs:
+  argocd-image-updater-version:
+    description: The version of ArgoCD Image Updater to install
+    required: true
+    default: latest
+  argocd-server:
+    description: ArgoCD server URL
+    required: false
+  argocd-token:
+    description: ArgoCD server authentication token
+    required: false
+outputs:
+  argocd-image-updater-version:
```

Comment:
This should never differ from the input should it? Is there a reason to output this?

Reply:
This will differ if the input is `latest` which is default. The output will be the real version of the installed binary